src/components/Toaster/Toast/Toast.tsx

```diff
@@ -202,6 +204,17 @@ export function Toast(props: ToastUnitedProps) {
         [type || 'default']: true,
     };
 
+    React.useEffect(() => {
+        if (onMount && heightProps.ref.current) {
+            onMount({props, element: heightProps.ref.current});
+        }
+        return () => {
+            if (onUnmount && heightProps.ref.current) {
+                onUnmount({props, element: heightProps.ref.current});
+            }
+        };
+    }, []);
```

Comment:
Why there are no `onMount`, `onUnmount` dependencies here?

Reply:
I don't want to do smth when `onMount` or `onUnmount` changes.
`onMount` and `onUnmount` should fired only once during corresponding stages.
Using  `React.useEffect` without debs is common pattern for this case as I know.
Actual value of these functions will be used when time comes.

Reply:
Ok. I'll leave it up to Alexey @ogonkov :)